src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27474.cs

```diff
@@ -0,0 +1,24 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue27474 : _IssuesUITest
+    {
+        public Issue27474(TestDevice testDevice) : base(testDevice)
+        {
+        }
+
+        public override string Issue => "Picker view items are not displayed when setting the title on the picker control";
+
+        [Test]
+        [Category(UITestCategories.Picker)]
+        public void ShouldDisplayPickerItemsWhenOpeningPicker()
```

Comment:
The test is failing on Mac because the reference snapshot is pending. Already available in the latest build.
![image](https://github.com/user-attachments/assets/1f1384c5-0810-487b-a208-b9534e72fee6)
Could you commit it?